gprofiler/utils/__init__.py

```diff
@@ -148,7 +148,9 @@ def start_process(
         else:
             # explicitly remove our directory from LD_LIBRARY_PATH
```

Comment:
Move the comment to its relevant code line now. And add another comment explaining the TMPDIR ("respect the TMPDIR we were run with in all cases")